packages/environment/test/develop.js

```diff
@@ -0,0 +1,67 @@
+const assert = require("chai").assert;
+var fs = require("fs-extra");
+var glob = require("glob");
+const { default: Box } = require("@truffle/box");
+const { Environment } = require("@truffle/environment");
+const sinon = require("sinon");
+
+describe("Environment develop", function () {
+  let config;
+
+  const expectedNetwork = {
+    port: 8545,
+    network_id: 1342,
+    gas: 90000
+  };
+
+  const testOptions = {
+    port: 5678,
+    network_id: expectedNetwork.network_id,
+    gas: 7000
+  };
+
+  before("Create a sandbox", async function () {
+    this.timeout(5000);
+    config = await Box.sandbox("default");
+    config.network = "network";
+    config.networks = {
+      network: {
+        port: expectedNetwork.port,
+        network_id: 4321,
+        gas: expectedNetwork.gas
+      }
+    };
+    sinon.stub(Environment, "detect").returns();
```

Comment:
If you are not concerned with having a return value, just delete the chained `.returns` here.

Reply:
ok, will remove the chained .returns